### DISCUSSION TITLE
cortx technical architecture and competitor analysis

### DISCUSSION THREAD

**Discussion by leon171** [Seagate/cortx#398] [Q&A] [answered]
Cortx looks a cool solution and I really want to know more details about it. But I am getting lost from high-level marketing post and numerous source code. There seems to be python/c++/rabittMQ. Could anyone know where to see a whole technical architecture of cortx to show components and relations?

Furthermore, from user perspective, cortx is still another object storage solution needing more ecosystem engagement. The why in FAQ did not give enough info about its strength vs others. Could anyone share some detailed architecture&function comparison between cortx and other open source solutions like Ceph/Gluster? 

Thanks a lot!

**Comment by amitwac2608** [comment]
Thanks @leon171 for your interest in CORTX. You can start with this doc: https://github.com/Seagate/cortx/blob/main/doc/be/CORTX-S3OVERVIEW.rst

In this document, you may find old terminologies (like mero is now motr, clovis is motrlib). You can refer other documents in the same directory and let us know if these documents helps you in understanding CORTX better.

**Reply by leon171** [reply]
thanks a lot, Amit. do you have hints for my second question?

**Reply by amitwac2608** [reply]
You can find CORTX comparison with its competitiors here : https://github.com/Seagate/cortx/blob/main/doc/be/CORTX_CORE_ARCHITECTURE.rst

If you want more detailed comparisons, I guess @johnbent, @bkirunge7  can connect you to right person/document.

**Reply by leon171** [reply]
Yes, I do want more detailed comparison between cortx vs. open source ones if possible. Thanks for your kindness.

**Comment by johnbent** [comment, marked as answer]
Hello, @leon171 , please also look at the cortx-motr repo.  There is a lot of architectural info in there.  Not all of it is linked yet though so you might also just browse the cortx-motr/doc/PDF folder: https://github.com/Seagate/cortx-motr/tree/main/doc/PDF

**Reply by leon171** [reply]
Thank you for all your help!

**Comment by yanqingfu** [comment]
maybe this video[ Intel and Seagate Deliver Next-Gen Complete Data Center Storage Platforms ](https://www.youtube.com/watch?v=gCybHC8TlB8&list=PLOLUar3XSz2M_w5OxQLNDBTpSrsGbjDWr&index=4) can also give you some ideas
